gravitl/netmaker-ui
# fix(GRA-1222): remove allow node signup w/o keys

TESTING STEPS:
1. Check that "allow node signup w/o keys" is no longer in UI
2. Check networks, nodes screens
